src/base.py

```diff
@@ -60,14 +33,13 @@ class BlockType(str, Enum):
     TABLE = "Table"
     FIGURE = "Figure"
     INFERRED = "Inferred from gaps"
-    GOOGLE = "Google Text Block"
-    AMBIGUOUS = "Ambiguous"  # TODO: remove this when OCRProcessor._infer_block_type is implemented
+    AMBIGUOUS = "Ambiguous"
+    GOOGLE_BLOCK = "Google Text Block"
 
```

Comment:
i think remove these and set default to `text`?

Reply:
@THOR300 this is the one outstanding comment, but in retrospect maybe we can't remove these if we're not rerunning the parser on all docs?

Reply:
These text blocks types are referenced via the data access lib now so we would have to make the change on [this PR](https://github.com/climatepolicyradar/data-access/blob/60beedd11e4d26979675eaf69632adb113dc0c7d/src/cpr_data_access/parser_models.py#L33)